komga-webui/src/locales/en.json

```diff
@@ -166,12 +165,12 @@
   },
   "browse_readlist": {
     "edit_elements": "Edit elements",
-    "edit_readlist": "Edit read list"
+    "edit_readlist": "Edit reading list"
   },
   "browse_series": {
-    "earliest_year_from_release_dates": "This is the earliest year from the release dates from all books in the series",
-    "series_no_summary": "This series has no summary, so we picked one for you!",
-    "summary_from_book": "Summary from book {number}:"
+    "earliest_year_from_release_dates": "Earliest year of any released book in this series.",
+    "series_no_summary": "This series has no summary, so one was randomly selected from one of the series books.",
```

Comment:
it is not picked randomly

Reply:
It was suggested here https://github.com/gotson/komga/pull/567/files#r777714590